app/assets/stylesheets/provider/_header.scss

```diff
@@ -1,11 +1,10 @@
 $header-height: line-height-times(63 / 20);
 $master-ribbon-height: line-height-times(1);
 
-.Header {
-
+header {
```

Comment:
I'd keep the `.Header` class and target it, because in the future we or any of our libraries may introduce more `<header>` tags. In fact, PF4/React introduces several of them, we already have some in the Login Page

Reply:
OK! Didn't know that